interfaces/builtin/ppp.go

```diff
@@ -0,0 +1,115 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+var pppConnectedPlugAppArmor = []byte(`
+# Description: Allow operating ppp daemon. Reserved because this gives
+#  privileged access to the ppp daemon.
+# Usage: reserved
+
+# Needed for modem connections using PPP
+/usr/sbin/pppd ix,
+/etc/ppp/** rwix,
+/dev/ppp rw,
+/dev/tty[^0-9]* rw,
+/run/lock/*tty[^0-9]* rw,
+/run/ppp* rw,
+/var/run/ppp* rw,
+/var/log/ppp* rw,
+/bin/run-parts ix,
+@{PROC}/@{pid}/loginuid r,
+capability setgid,
+capability setuid,
+/sbin/resolvconf rix,
+/run/resolvconf** rw,
+/etc/resolvconf/** rw,
+/etc/resolvconf/update.d/* ix,
+/lib/resolvconf/* ix,
+`)
```

Comment:
I'm mildly surprised that there isn't a ConnectedPlugSeccomp, but hey, I'm not complaining! :)